controllers/operator_controller.go

```diff
@@ -193,11 +229,6 @@ func (r *OperatorReconciler) generateExpectedBundleDeployment(o operatorsv1alpha
 			},
 		},
 	}
-
-	if err := ctrl.SetControllerReference(&o, dep, r.Scheme); err != nil {
```

Comment:
Does this need to be done anywhere else?

Reply:
Looks like it's being done manually above on 196. Follow-up though: what's the reason behind doing it manually vs using the `ctrl.SetControllerReference` func?

Reply:
Avoid having to handle an error.